airflow/exceptions.py

```diff
@@ -372,9 +372,15 @@ class TaskDeferred(BaseException):
     Signal an operator moving to deferred state.
 
     Special exception raised to signal that the operator it was raised from
-    wishes to defer until a trigger fires.
+    wishes to defer until a trigger fires. Triggers can send execution back to task or end the task instance
+    directly. If the trigger will end the task instance itself, ``method_name`` should be
+    None; otherwise, provide the name of the method that should be used when
+    resuming execution in the task.
```

Comment:
```suggestion
    wishes to defer until a trigger fires. Triggers can send execution back to task or end the task instance
    directly. If the trigger should end the task instance itself, ``method_name`` should be
    None; otherwise, provide the name of the method that should be used when
    resuming execution in the task.
```

Reply:
hmm, well does the method name even matter in this case? my guess is it is ignored?

Reply:
Yeah I think it does not matter, and the docstring should just say so instead.

Reply:
I have added the same wording.